cmd/demo/demo_test.go

```diff
@@ -3,19 +3,16 @@
 // perun-eth-demo. Use of this source code is governed by the Apache 2.0
 // license that can be found in the LICENSE file.
 
-// +build on_chain_eth_test
```

Comment:
Great fix! We should add something to the `README.md` on how to start the the test so the user can try it.  
Have you been testing it with instant mining or blocktime 5s? Does not work for me with 5s blocktime so we should add a specific `ganache-cli` command for the test.

Reply:
Maybe also reference the `(alice|bob).log` files so that the user can find the error, in case he setup something wrong.

Reply:
I have been testing with blocktime 5s and it works for me.

I don't think we need to guide the user through the testing. The tests are for us, not for the demo users, right?

Reply:
I get a `Could not initialize node." error="setting up contracts: contract setup failed: validating asset holder: incorrect contract code: invalid bytecode at address` error currently in `bob.log`…